docs/troubleshooting.md

```diff
@@ -22,3 +22,4 @@ discusses potential solutions to the same.
 | Unable to unmount or stop GCSFuse due to an error message like:`fusermount: failed to unmount: Device or resource busy` or `umount: /path/to/mountpoint: target is busy`.</br>This typically indicates active processes are using files or directories within the GCSFuse mount.                                                                                                                                                                                                                                                                                                                                      | Find the process ID of GCSFuse:<br/>`BUCKET=<Enter your bucket name>`</br>` MOUNT_POINT=<Enter your mount point>`</br>`PID=$(ps -aux &#124; grep "gcsfuse.*$BUCKET.*$MOUNT_POINT" &#124; grep -v grep &#124; tr -s ' ' &#124; cut -d' ' -f2)`</br>Kill the GCSFuse process:</br>`sudo kill -SIGKILL "$PID"`</br>Unmount GCSFuse</br>`fusermount -u $"MOUNT_POINT"`                                                                                                                                                                                                                                                     |
+| ls: reading directory `<mountpath>/<path>`:  Input/output error                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | Find out if you have any object(s) with name/prefix having `//` in it or starting with `/`, in the mounted GCS bucket (use `gsutil ls gs://<bucket>/<path>` to find out). If yes, move/rename (use `gsutil mv -r`) such objects to name/prefix not having `//` or starting with `/`, or delete it (use `gsutil rm -r`). Refer [semantics](semantics.md#unsupported-object-names) for more details.                                                                                                                                                                                                                     |
```

Comment:
could we add the full gsutil command to mitigate this. That was useful. gsutil mv A//B to A/B

Reply:
done